asmcli/asmcli

```diff
@@ -1597,11 +1601,12 @@ install_private_ca() {
   # If modify_gcp_component permissions are not granted, it is assumed that the
   # user has taken care of this, else Istio setup will fail
   local CA_NAME; CA_NAME="$(context_get-option "CA_NAME")"
+  local CA_POOL_URL; CA_POOL_URL=$(echo "${CA_NAME}" | cut -f1 -d:)
   local FLEET_ID; FLEET_ID="$(context_get-option "FLEET_ID")"
```

Comment:
nit: maybe add an additional check to see if there is more than one ":' in the string, this errors out as an unacceptable format.

Reply:
Added in validate_private_ca().